frontend/packages/kubevirt-plugin/integration-tests/tests/mocks/mocks.ts

```diff
@@ -26,15 +26,15 @@ import {
 
 export const provisionSources = {
   [ProvisionSource.URL]: {
-    method: ProvisionSource.URL,
+    method: ProvisionSourceItem.URL,
     source: 'https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img',
   },
   [ProvisionSource.CONTAINER]: {
-    method: ProvisionSource.CONTAINER,
+    method: ProvisionSourceItem.CONTAINER,
     source: 'kubevirt/fedora-cloud-registry-disk-demo:latest',
   },
-  [ProvisionSource.PXE]: { method: ProvisionSource.PXE },
-  [ProvisionSource.DISK]: { method: ProvisionSource.DISK },
+  [ProvisionSource.PXE]: { method: ProvisionSourceItem.PXE },
```

Comment:
what benefit do we get from splitting ProvisionSource into 2 constants? This will result in worse maintainability.

Can we either

1. rename the original to what we need: `URL (adds disk)`, etc.
2. refactor the ProvisionSource to become ObjectEnum so it has all the required information in one place (name, label and source)

Reply:
I added `ProvisionSourceItem` because string like `URL (adds disk)` cannot be used as the key.
Will look into `ObjectEnum`.

Reply:
Why not? Is there a specific reason why it cannot be used as a key?

Reply:
If using `URL (adds disk)` , it throws errors very quickly:
```
[10:33:24] E/launcher - Error: TypeError: Cannot read property 'source' of undefined
    at Object.exports.getTestDataVolume (/home/gouyang/console/frontend/packages/kubevirt-plugin/integration-tests/tests/mocks/mocks.ts:102:37)
```

Reply:
@suomiy Could you give me an example how to use ObjectEnum?